### PR TITLE
Use first config file found in a list of paths…

### PR DESCRIPTION
**Pull request by bolmar**
… allowing falling back to a global config file.

### CODE REVIEW

File: src/els_config.erl [hunk 1]
```diff
@@ -21,7 +21,6 @@
 %%==============================================================================
 %% Macros
 %%==============================================================================
--define(DEFAULT_CONFIG_PATH, "erlang_ls.config").
```
Comment (jfacorro) [inline src/els_config.erl]: After this change the code is still using this constant, so it would be nice to keep it.

File: README.md [hunk 2]
```diff
@@ -252,9 +252,31 @@ Code completion is available for the following elements:
...
+* <project root>/erlang_ls.config
+* <app config path>/erlang_ls/erlang_ls.config
+
+Generally speaking, <app config path> should be:
+
+| Operating System | Configuration Directory                             |
+|------------------|-----------------------------------------------------|
+| Linux            | /home/USER/.config/erlang\_ls                       |
+| OS X             | /Users/USER/Library/Application\ Support/erlang\_ls |
+| Windows          | c:/Users/USER/AppData/Local/erlang\_ls              |
```
Comment (alanz) [inline README.md]: The `<app config path>` in the table already includes the `erlang_ls` directory, I presume it will not be repeated?

**Comment by robertoaloi** [comment]
> Would it make sense to load the config files in order, from the default, to the user, to the project, with each replacing what is already there.
> 
> This allows "general" settings, which can be overridden at the project level as needed.

It's an interesting idea, but I guess it could be implemented separately from this PR. Maybe just open a ticket for this?

**Comment by robertoaloi** [comment]
Agreed with @alanz to merge this.

**Pull request merged**